docs/services/vision/detection.md

```diff
@@ -213,32 +213,26 @@ Add the Vision Service object to the services array in your raw JSON configurati
 {{% /tab %}}
 {{< /tabs >}}
 
-Click **Save config** and head to the **Components** tab.
+Click **Save config**.
 
-## Add a camera component and a "transform" model
+## Test your detector
 
-You cannot interact directly with the [Vision Service](/services/vision/).
-To be able to interact with the Vision Service you must:
+You can test your detector with [live camera footage](#live-camera-footage) or [existing images](#existing-images).
 
-1. Configure a physical [camera component](../../../components/camera/).
-Pass the name of this camera to Vision Service methods.
-2. (Optional) To view output from the detector overlaid on images from the physical camera, configure a [transform camera](../../../components/camera/transform/).
+### Live Camera footage
 
-    After adding the component and its attributes, click **Save config**.
+If you intend to use the detector with a camera that is part of your robot, you can test your detector from the [**Control tab**](/manage/fleet/robots/#control) or with code:
 
-    Wait for the robot to reload, and then go to the **Control** tab and open the transform camera card to test the stream of detections.
+1. Configure a physical [camera component](../../../components/camera/).
+2. If you would like to see detections from the **Control tab**, configure a [transform camera](../../../components/camera/transform/) to view output from the detector overlaid on images from the physical camera.
+   If you only want to access detections from code, you do not need to configure a transform camera.
+3. After adding the components and their attributes, click **Save config**.
+4. Navigate to the **Control** tab, click on your transform camera and toggle it on.
+   The transform camera will now show detections with bounding boxes around the object.
 
 ![Viam app control tab interface showing bounding boxes around two office chairs, both labeled "chair" with confidence score "0.50."](/services/vision/chair-detector.png)
 
-## Code
-
-The following code gets the robot’s vision service and then runs a color detector vision model on output from the robot's camera `"cam1"`.
-
-{{% alert title="Tip" color="tip" %}}
```

Comment:
Why get rid of this alert? I added it because of the user feedback (via Fahmina) that this is a common mistake

Reply:
Perhaps my line 235 suggestion is a way to address this.

Reply:
Because it doesn't have to be a physical camera - you could have a transform camera that crops the image and then pass that. So specifically calling out that it is not the name of the transform camera is weird.

Reply:
I think the thing that needs to be emphasized is that when setting up the vision service on python, you shouldn't call "get_classifications" or "get_detections" using the camera that already has the "detections" or "classifications" transform applied to it -- it would be like calling "Detections" on a camera that already called "Detections"

Reply:
So yea, if your transform camera just is doing a crop, then using get_detections_from_camera on it is fine, but I do remember that people have made a mistake of sometimes creating a transform camera that applies detections, and then in python, using that camera to call get_detections again (which creates a weird output and a slow camera)

Reply:
Ok - I've aded that as an alert - I like that way of putting it